README.md

```diff
@@ -24,51 +20,45 @@ Typeshed fully supports Python versions 3.8 and up. Support for Python 3.7
 is limited: see https://github.com/python/typeshed/issues/10113
 for details.
 
+---
+
 ## Using
 
 If you're just using a type checker ([mypy](https://github.com/python/mypy/),
 [pyright](https://github.com/microsoft/pyright),
-[pytype](https://github.com/google/pytype/), PyCharm, ...), as opposed to
-developing it, you don't need to interact with the typeshed repo at
-all: a copy of standard library part of typeshed is bundled with type checkers.
-And type stubs for third party packages and modules you are using can
-be installed from PyPI. For example, if you are using `six` and `requests`,
-you can install the type stubs using
+[pytype](https://github.com/google/pytype/) or PyCharm, ...), and not developing it, you do not need to interact directly with the typeshed repository. A copy of the standard library portion of typeshed is typically bundled with type checkers. Additionally, you can install type stubs for third-party packages and modules from PyPI. For instance, if you are using `six` and `requests`, you can install the type stubs using the following command:
 
 ```bash
 $ pip install types-six types-requests
 ```
 
-These PyPI packages follow [PEP 561](http://www.python.org/dev/peps/pep-0561/)
+These PyPI packages adhere to [PEP 561](http://www.python.org/dev/peps/pep-0561/)
 and are automatically released (multiple times a day, when needed) by
 [typeshed internal machinery](https://github.com/typeshed-internal/stub_uploader).
 
 Type checkers should be able to use these stub packages when installed. For more
 details, see the documentation for your type checker.
 
+---
 ### The `_typeshed` package
 
 typeshed includes a package `_typeshed` as part of the standard library.
-This package and its submodules contains utility types, but is not
+This package and its submodules contain utility types but are not
```

Comment:
You're correct that the grammar isn't spot-on here, but I don't think it's a major issue, really. And if anything, your PR makes it worse. If we're being picky about this, it should probably be something like

```md
This package and its submodules contain utility types, but the package is not
```